src/api/app/assets/stylesheets/webui2/user.scss

```diff
@@ -1,3 +1,34 @@
 #home-login {
   color: $gray-500;
 }
+
+.activity-percentil0 {
+  background-color: #dcd5da;
```

Comment:
I have nothing against pink, but it doesn't fit in the UI. I suggest using green, so something similar to the `$obs_green` variable we have in [src/api/app/assets/stylesheets/webui2/bootstrap_variables/colors.scss](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/stylesheets/webui2/bootstrap_variables/colors.scss).

Reply:
if everything is green, the colour has no more meaning.

Reply:
It's so random, but whatever. Let's use purple for the next feature!

Reply:
Well, the pink kind of makes it obvious that obs needs a designer - but just picking $obs_green is wrong as well.

Reply:
It is not random : angry: the more you contribute the more pink you get! :pink_heart:

Reply:
It's fine, I'll just change it with Stylus: https://addons.mozilla.org/en-US/firefox/addon/styl-us/